dashboard/lib/tasks/seed.rake

```diff
@@ -97,10 +97,40 @@ namespace :seed do
   end
 
   task school_districts: :environment do
+    # use a much smaller dataset in environments that reseed data frequently.
+    school_districts_tsv = CDO.stub_school_data ? 'test/fixtures/school_districts.tsv' : 'config/school_districts.tsv'
+    expected_count = `wc -l #{school_districts_tsv}`.to_i - 1
+    raise "#{school_districts_tsv} contains no data" unless expected_count > 0
+
     SchoolDistrict.transaction do
-      # Since other models (e.g. Pd::Enrollment) have a foreign key dependency
-      # on SchoolDistrict, don't reset_db first.  (Callout, above, does that.)
-      SchoolDistrict.find_or_create_all_from_tsv!('config/school_districts.tsv')
+      # It takes approximately 30 seconds to seed the school districts data from tsv.
+      # Skip seeding if the data is already present. Note that this logic may need
+      # to be updated once we incorporate data from additional survey years.
+      if SchoolDistrict.count < expected_count
+        # Since other models (e.g. Pd::Enrollment) have a foreign key dependency
+        # on SchoolDistrict, don't reset_db first.  (Callout, above, does that.)
+        puts "seeding school districts (#{expected_count} rows)"
+        SchoolDistrict.find_or_create_all_from_tsv!(school_districts_tsv)
+      end
+    end
+  end
+
+  task schools: :environment do
+    # use a much smaller dataset in environments that reseed data frequently.
+    schools_tsv = CDO.stub_school_data ? 'test/fixtures/schools.tsv' : 'config/schools.tsv'
+    expected_count = `wc -l #{schools_tsv}`.to_i - 1
+    raise "#{schools_tsv} contains no data" unless expected_count > 0
+
+    School.transaction do
+      # It takes approximately 4 minutes to seed the schools data from tsv.
+      # Skip seeding if the data is already present. Note that this logic may need
+      # to be updated once we incorporate data from additional survey years.
```

Comment:
Might want to say it takes 4 minutes to seed the full set. The fixture for non-prod environments is *much* smaller/faster (nearly immediate). Same above.